### ISSUE TITLE
UI update in sign in / sign up page

### ISSUE THREAD

**Issue by ayushrana83** [AkshitLakhera/PenCraft-Full-Stack-Blogging-Application#49]
In dark mode, the text is hard to read.
![Screenshot 2024-10-03 213422](https://github.com/user-attachments/assets/0697f591-cecb-4dbb-91f4-c58da7ce6a45)

**Comment by udayempire** [comment]
@ayushrana83 Hello I would love to contribute to this project. Please let me know if I can continue.

**Comment by iamDyeus** [comment]
I suggest you guys use some auth UI from [FarmUI](https://www.farmui.com/). Check [this](https://www.farmui.com/example/farmui-auth-02) out

**Comment by udayempire** [comment]
.take

**Comment by AkshitLakhera** [comment]
> I suggest you guys use some auth UI from [FarmUI](https://www.farmui.com/). Check [this](https://www.farmui.com/example/farmui-auth-02) out

Yes ,we can use these ui components, @ayushrana83  can you implement this ?

**Comment by ayushrana83** [comment]
Sure, I can try. 
What content should we put on the left side?

**Comment by AkshitLakhera** [comment]
> Sure, I can try. What content should we put on the left side?

see for now put any dummy content , or you  can find any content from gpt, I in future I  will update it.

**Comment by ayushrana83** [comment]
This is how it looks . Is there any changes needed ?
![in](https://github.com/user-attachments/assets/a23394ee-e522-42cc-9909-c0e1c8fd9b4b)
![up](https://github.com/user-attachments/assets/5739dd8f-9d3a-4363-bacd-25820c9384b4)

**Comment by AkshitLakhera** [comment]
it looks good make pr ,then I will review code .If needed I will suggest some add on.

**Comment by iamDyeus** [comment]
@ayushrana83 i suggest you use this for both signin and signup (change the input fields for signin in this layout)

<br/>

![image](https://github.com/user-attachments/assets/bb9566ba-d52c-4e40-9fb5-289787f100b3)
